tests/specification/dummy.wast

```diff
@@ -0,0 +1,95 @@
+;; i32 operations

```

Comment:
Please add copyright

```rs
/*
# This file incorporates code from the WebAssembly testsuite, originally
# available at https://github.com/WebAssembly/testsuite.
#
# The original code is licensed under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance
# with the License. You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
*/
```

Reply:
Maybe I should remove `dummy.wast`? It's there more for a playground for us to copy-paste code into and play with it. What do you (and others) think about it?

Reply:
I think removing it is sensible, if we already have the submodule repo with the entire wast body.

Reply:
removed